src/main/kotlin/com/terraformation/backend/accelerator/model/ExternalApplicationStatus.kt

```diff
@@ -0,0 +1,46 @@
+package com.terraformation.backend.accelerator.model
+
+import com.terraformation.backend.db.accelerator.ApplicationStatus
+
+enum class ExternalApplicationStatus {
+  NotSubmitted,
+  FailedPreScreen,
+  PassedPreScreen,
+  InReview,
+  Accepted,
+  NotAccepted,
+  Waitlist;
+
+  fun shouldNotify(): Boolean {
```

Comment:
This is fine, but an alternate approach would be to make `shouldNotify` a property of the enum class and set it as part of the value declarations, e.g.,
```
enum class ExternalApplicationStatus(val shouldNotify: Boolean) {
  NotSubmitted(false),
  ...
}
```